packages/devtools_app/lib/src/config_specific/theme_overrides/theme_overrides_web.dart

```diff
@@ -0,0 +1,81 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be found
+// in the LICENSE file.
+
+import 'dart:html';
+
+import 'package:flutter/widgets.dart';
+
+/// Environment-specific theme overrides, for example IDE-provided
+/// theming for embedded mode.
+class ThemeOverrides {
+  ThemeOverrides({this.backgroundColor, this.foregroundColor, this.fontSize});
+
+  factory ThemeOverrides.load() {
+    final queryParams = _loadQueryParams();
+
+    final overrides = ThemeOverrides(
+      backgroundColor: _tryParseColor(queryParams['backgroundColor']),
+      foregroundColor: _tryParseColor(queryParams['foregroundColor']),
+      fontSize: _tryParseDouble(queryParams['fontSize']),
+    );
+
+    // If the environment has provided a background color, set it immediately
+    // to avoid a white page until the first Flutter frame is rendered.
+    if (overrides.backgroundColor != null) {
```

Comment:
for this to work we need to also make sure that the main dart.js script is loaded synchronously. That should be fine to enforce as DevTools isn't at all useful until the script is loaded as we aren't a PWA.

Reply:
It's currently imported in `packages/devtools_app/web/index.html` just before the closing `</body>` without any async/defer tag:

```
<script src="main.dart.js" type="application/javascript"></script>
```

Is that correct?